FrontEnd/src/components/Dropzone.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { parse, ParseConfig } from 'papaparse';
+import Box from '@mui/material/Box';
+import FileUploadIcon from '@mui/icons-material/FileUpload';
+import { Typography } from '@mui/material';
+import { Stack } from '@mui/material';
+
+export interface DropzonePureProps {
+  isDragActive: boolean;
+  getRootProps: any;
+  getInputProps: any;
+}
+
+export function DropzonePure({
+  getRootProps,
+  getInputProps,
+  isDragActive,
+}: DropzonePureProps) {
+  return (
+    <Box
+      component={Stack}
+      direction="column"
+      justifyContent="center"
+      sx={{
+        width: 700,
+        height: 400,
+        backgroundColor: '#E2EAE6',
+        m: 'auto',
+        mt: 8,
+        border: '3px dashed',
+        borderColor: 'primary.main',
+      }}
+      {...getRootProps()}
+    >
+      <Typography align="center">
+        {' '}
+        <FileUploadIcon color="primary" sx={{ fontSize: 80 }} />{' '}
+      </Typography>
+      <input {...getInputProps()} />
+      {isDragActive ? (
+        <Typography color="primary" align="center">
+          Drop file here...
+        </Typography>
+      ) : (
+        <Typography color="primary" align="center">
+          <b>Choose a CSV file</b> or drag it here.
+        </Typography>
+      )}
+    </Box>
+  );
+}
+
+export default function Dropzone() {
+  // @ts-ignore
+  const onDrop = useCallback((acceptedFiles) => {
+    acceptedFiles.forEach((file: File) => {
+      if (file.type !== 'text/csv') {
+        alert(
+          'Only .csv files are supported. Uploaded file is called:' +
```

Comment:
One last thing! When I tried building this I got the following error:

![image](https://user-images.githubusercontent.com/22805017/174008943-5a2c43a1-3eb5-4050-8433-aa189768f414.png)

Try cloning the repo to see if you can produce this error. Then, try the following to fix this issue (which worked for me) then push the change:

`npx prettier --write src/components/Dropzone.tsx`